src/states/elder/mod.rs

```diff
@@ -621,14 +645,6 @@ impl Elder {
             return Err(RoutingError::UnknownConnection(pub_id));
         };
 
-        if self.chain.is_peer_our_member(&pub_id) {
```

Comment:
just would like to know why this check can be skipped now?

Reply:
I had some reason to remove it but I don't think it's valid anymore. I'll put it back.